src/gawronski-bartosz/Fsd.Bartek.Cs/Fsd.Bartek.Ex4.Services/Sql/SqlConnect.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Fsd.Bartek.Ex4.Services.Services;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Fsd.Bartek.Ex4.Data.Entities;
+using Fsd.Bartek.Ex4.Data.Enums;
+
+namespace Fsd.Bartek.Ex4.Services.Sql
+{
+    public class SqlConnect : IProductsService
+    {
+        string connectionString = @"Server=.\BARTEK;Database=Shop;User Id=sa;Password=restVen;";
```

Comment:
sposób połączenia z bazą będzie uniwersalny dla każdej tabeli, tak samo jak sposób zapytania. WIęc najlepiej zrobić klasę bazową, która będzie miała takie uniwersalne metody (get, save, delete). Jak za chwile poprosze o dpisanie serwisu dla użytkowników to nie będzie już problemu